Server/Content/Sources/App/configure.swift

```diff
@@ -0,0 +1,24 @@
+import Fluent
+import FluentPostgresDriver
+import Vapor
+
+// configures your application
+public func configure(_ app: Application) throws {
+    // uncomment to serve files from /Public folder
+    // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory))
+
+    app.databases.use(.postgres(
+        hostname: Environment.get("DATABASE_HOST") ?? "localhost",
+        username: Environment.get("DATABASE_USERNAME") ?? "vapor_username",
+        password: Environment.get("DATABASE_PASSWORD") ?? "vapor_password",
+        database: Environment.get("DATABASE_NAME") ?? "vapor_database"
```

Comment:
Serve environment variables from `.env` file and use more appropriate & secure password and do not add it to git